### PR TITLE
fix: pop up menu

### PR DESCRIPTION
**Pull request by gjulivan**
Fix pop up menu changelog and versioning.
Fix z-index on datagrid filter causing popup menu showing on top of the filter while the filter in sticky position.

related to [WC-1125] and [WC-1705]
and related PR: https://github.com/mendix/web-widgets/pull/545/
https://github.com/mendix/atlas/pull/49

[WC-1125]: https://mendix.atlassian.net/browse/WC-1125?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[WC-1705]: https://mendix.atlassian.net/browse/WC-1705?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by iobuhov** [comment]
E2E failed because
```
Error: [@mendix/popup-menu-web] package.json and package.xml versions do not match.
```

**Review by iobuhov — APPROVED** [review]

File: packages/modules/data-widgets/CHANGELOG.md [hunk 1]
```diff
@@ -6,8 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
-### Fixed
+#### Fixed
 
+-   We fixed popup menu showing on top of datagrid filter in sticky position.
 -   We removed dead span styling from module SCSS
```
Comment (iobuhov) [inline packages/modules/data-widgets/CHANGELOG.md:12]:
```suggestion
### Fixed

-   We fixed popup menu showing on top of datagrid filter in sticky position.

-   We removed dead span styling from module SCSS
```

four hash (####) sction is for "nested" widgets
three hash (###) section is for "data-widgets" changes

**Review by iobuhov — CHANGES_REQUESTED** [review]

**Review by iobuhov — APPROVED** [review]

**Review by samuelreichert — APPROVED** [review]
LGTM

File: packages/pluggableWidgets/popup-menu-web/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Fixed
+
+-   We fixed an issue causing popup menu to appear behind other elements(modal popups). This fix will move popupmenu-menu wrapper DOM in the same level as the trigger button. Thus, custom styling targeting the popup menu might affected.
```
Comment (r0b1n) [inline packages/pluggableWidgets/popup-menu-web/CHANGELOG.md:11]: Suggested:
```
We have addressed an issue that was causing the popup menu to appear behind other elements, such as modal popups. This fix involved modifying the DOM structure by moving the `popupmenu-menu` element to the same level as the trigger button. As a result, custom styling that targets the popup menu may have been affected.
```

File: packages/modules/data-widgets/src/themesource/datawidgets/web/_datagrid.scss [hunk 3]
```diff
@@ -221,7 +221,7 @@ $grid-selected-row-background: $brand-light;
         }
 
         > .td-text,
-        * {
+        > * {
```
Comment (iobuhov) [inline packages/modules/data-widgets/src/themesource/datawidgets/web/_datagrid.scss:224]: Risky, but I happy that we fix this selector.

File: packages/pluggableWidgets/popup-menu-web/src/ui/PopupMenu.scss [hunk 4]
```diff
@@ -1,6 +1,11 @@
 .popupmenu {
     position: relative;
     display: inline-flex;
+    overflow: visible;
+}
+
+.table .td > .popupmenu {
```
Comment (iobuhov) [inline packages/pluggableWidgets/popup-menu-web/src/ui/PopupMenu.scss:7]: This is bit overkill, now we have styles that depends on other widgets. Is there other way to fix it?

**Review by leonardomendix — APPROVED** [review]

**Review by iobuhov — APPROVED** [review]
LGTM

**Pull request merged**